recipes/cclib/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cclib" %}
+{% set version = "1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 636dfbc192c0a95079aaba3fb0c4872222c2a8d500f28d681bc0522fe81dc49a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py26]
+  skip: True  # [py33]
```

Comment:
These two skips are not needed since conda-forge does not support those versions of Python.